bulk_grades/api.py

```diff
@@ -357,12 +357,13 @@ def preprocess_row(self, row):
         Preprocess the CSV row.
         """
         operation = {}
-        if row['user_id'] in self._users_seen:
-            return operation
+        user_id = row['user_id']
+        if user_id in self._users_seen:
+            raise ValidationError(_(f'Repeated user_id {str(user_id)}'))   # pylint: disable=translation-of-non-string
```

Comment:
I believe this can be solved by just pulling the user id out of the translation call [`_()`] ?
i.e. `ValidationError(_('Repeated user_id:  ') + str(user_id)})`